HW0/P5/P5.txt

```diff
@@ -1 +1,20 @@
 # Your answers here.
+1. 255s
+
+2. Yes, they only need 34s.
+
+3. 8s
+
+4. log2(N)
+
+5. 15s for infinite employess.
+
+6. Takes one bag:  272s for infinite employees
```

Comment:
It seems that you could benefit from overlapping communication and computation, so the employees can start counting and summing the bags (although not all of them got their bags).